pages/linux/microcom.md

```diff
@@ -0,0 +1,11 @@
+# microcom
+
+> A minimalistic terminal program.
+
+- Open a serial port using the specified baudrate:
+
+`microcom --port {{path/to/serial-port}} --speed {{baudrate}}`
```

Comment:
serial-port => serial_port
baudrate => baud_rate

Reply:
> "Use snake_case for multi-word tokens."

Jep, my bad. It's even in the CONTRIBUTING.md file, but I've skimmed right over it. Fixed.